### DISCUSSION TITLE
Software tools for teachers?

### DISCUSSION THREAD

**Discussion by thistlillo** [community/community#34051] [GitHub Education]
Last year, after proving my "teacher" status, I was able to download some software packages. Few days ago I re-verified my status, but could not find the pages with links to the software.

Two questions: is it gone? What about the life-licences that some of us got?

Is an explanation of what happened? I know I am not speaking of a constitutional right, but some words would be appreciated.

**Comment by thomasjohnson32** [comment]
can you please guide me I am a little bit confused.
I create the tool related to the printer to check the ink & I need to attach that tool to my website here is the link https://printercool.com/
Do I need to convert the tool into a plugin? manually in need to integrate get. 
looking forward to you.
